aws/resource_aws_cloudfront_distribution.go

```diff
@@ -451,7 +451,8 @@ func resourceAwsCloudFrontDistribution() *schema.Resource {
 									},
 									"restriction_type": {
 										Type:     schema.TypeString,
-										Required: true,
+										Optional: true,
+										Default:  "none",
```

Comment:
Nitpick: we should use the SDK provided constant here: `Default: cloudfront.GeoRestrictionTypeNone`

Reply:
Fair Nitpick. Updated.